api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddTeacherRoleToPersonGenerator.java

```diff
@@ -26,10 +26,10 @@ String getRoleType() {
 	@Override
     FieldOptions getRoleActivityFieldOptions(VitroRequest vreq) throws Exception {
 		return new ConstantFieldOptions(
-          "", "Select one",
-          "http://purl.org/ontology/bibo/Conference", "Conference", 
-          "http://vivoweb.org/ontology/core#Course",  "Course",
-          "http://purl.org/ontology/bibo/Workshop",   "Workshop");		
+		          "", "Select one",
+		          "http://purl.org/ontology/bibo/Conference", "Conference", 

```

Comment:
Please remove whitespace changes.

Reply:
Sorry that file was among those to change. I don't know how it ended up being comitted with those spaces.
Anyway I just did a new commit.